lib/parsing_nesting/tree.rb

```diff
@@ -115,7 +115,7 @@ def build_local_params(hash = {}, force_deftype = "dismax")
 
       if !hash.empty?
         defType = hash.delete(:defType) || hash.delete("defType")
-        "{!" + (defType ? "#{defType} " : "") + hash.collect { |k, v| "#{k}=#{v.to_s.include?(" ") ? "'" + v + "'" : v}" }.join(" ") + "}"
+        "{!" + (defType ? "#{defType} " : "") + hash.collect { |k, v| "#{k}=#{v.to_s.include?(" ") ? "'#{v}'" : v}" }.join(" ") + "}"
```

Comment:
Thanks for the contribution @martinlovell!  Sorry that nobody has gotten around to this sooner.  I see that the buils is passing now.

Would it be possible to submit a test that illustrates the issue that this solves?

My initial impression is a bit of nervousness around using interpolation as a strategy for handling strings vs arrays in this case.

Reply:
I'll try to put together some tests.  Thank you for the feedback.